hack/LiiklusService.proto

```diff
@@ -0,0 +1,139 @@
+syntax = "proto3";
+
+package com.github.bsideup.liiklus;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option java_package = "com.github.bsideup.liiklus.protocol";
+option optimize_for = SPEED;
+option java_multiple_files = true;
+
+option go_package = "liiklus";
+
+service LiiklusService {
```

Comment:
I'm pretty sure you can include only the rpc endpoints you need for the integration, to reduce the size of the generated code and self-document the usage

Reply:
It's far easier to just grab the latest protobuf definition for liiklus and not worry about what is needed and what is not when the file will be updated IMO.